include/openmc/constants.h

```diff
@@ -237,7 +237,8 @@ enum ReactionType {
   N_AC = 849,
   N_2N0 = 875,
   N_2NC = 891,
-  HEATING_LOCAL = 901
+  HEATING_LOCAL = 901,
+  PHT = 1001
```

Comment:
Rather than putting this under `ReactionType`, I would have a new entry under the `TallyScore` enum, perhaps called `SCORE_PULSE_HEIGHT`. The value is arbitrary so you can use the next negative integer, -17.

Reply:
Yes, that's right, I changed it in a commit accordingly.